glotaran/analysis/optimize.py

```diff
@@ -38,6 +38,8 @@ def optimize(scheme, verbose=True, client=None):
 
 def optimize_task(initial_parameter, scheme, verbose):
 
+    scheme.prepare_data()
```

Comment:
Yes, because I want to have the dataset svd as part of the result object. The copy is in principle unnecessary. Actually under the hood I think xarray already prevents copying here.

Reply:
The way I see it, as long as you ensure that your dataset contains the SVD object prior to analysis you can prevent the additional SVD computation during the call to optimize. Only with very large datasets (e.g. FLIM or stack of temporal scans) would doing the SVD computation at all become a bit of an annoying. Probably in >95% use cases it is a non issue, but perhaps we should make it optional at some point.